model/qti/Item.php

```diff
@@ -175,12 +175,12 @@ protected function getUsedAttributes()
     {
         return [
             'oat\\taoQtiItem\\model\\qti\\attribute\\Title',
-            'oat\\taoQtiItem\\model\\qti\\attribute\\Label',
-            'oat\\taoQtiItem\\model\\qti\\attribute\\Lang',
             'oat\\taoQtiItem\\model\\qti\\attribute\\Adaptive',
             'oat\\taoQtiItem\\model\\qti\\attribute\\TimeDependent',
+            'oat\\taoQtiItem\\model\\qti\\attribute\\Lang',
```

Comment:
@shpran Can you explain why this change was needed, please?

Reply:
When we are trying to update (save) item - we validate DOM content before processing and `<assesmentItem>` node attributes order is:
* adaptive
* timeDependent
* xml:lang
* label
**Example:**
```
<assessmentItem
  xmlns="http://www.imsglobal.org/xsd/imsqti_v2p2"
  xmlns:m="http://www.w3.org/1998/Math/MathML"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://www.imsglobal.org/xsd/imsqti_v2p2 http://www.imsglobal.org/xsd/qti/qtiv2p2/imsqti_v2p2.xsd"
  identifier="i5ea29a5d4e119112935c8357c06037d389"
  title="Aperçu"
  adaptive="false"
  timeDependent="false"
  xml:lang="fr-FR"
  label="2 Aperçu"
  toolName="TAO"
  toolVersion="3.4.0-sprint143"
>
```

When we are trying to preview item/booklet/etc. - we validate processed DOM content, and in this case the order is:
* label
* xml:lang
* adaptive
* timeDependent
**Example:**
```
<assessmentItem
  xmlns="http://www.imsglobal.org/xsd/imsqti_v2p2"
  xmlns:m="http://www.w3.org/1998/Math/MathML"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://www.imsglobal.org/xsd/imsqti_v2p2 http://www.imsglobal.org/xsd/qti/qtiv2p2/imsqti_v2p2.xsd"
  identifier="i5ea29a5d4e119112935c8357c06037d389"
  title="Aperçu"
  label="2 Aperçu"
  xml:lang="fr-FR"
  adaptive="false"
  timeDependent="false"
  toolName="TAO" toolVersion="3.4.0-sprint143"
>
```

To avoid unnecessary validation - I'm trying to compare stored content with provided. But in our case they will be fully similar except this attributes order. 
In this case we need to fix this order to make stored content and validated content similar.

Reply:
I got it now! Thanks!

Reply:
@shpran Another question about this. Seems that the order of the elements is really important and can cause issues as you described. 

Do not you think this should be explicitly declared so any new attribute added or refactoring on this code will not break it again? (I can say for instance some developer might be tempted to order these fields in alphabetical order - which will break again).

I mean, like a specialised common validation that sort the fields before validating, so it is explicit that the order matters.

I see you fix the problem here, but it wont avoid the same problem in the future. Wdyt?